HingeAngle/src/main/java/com/microsoft/device/display/samples/hingeangle/HingeAngleActivity.kt

```diff
@@ -34,22 +46,23 @@ class HingeAngleActivity : AppCompatActivity(), ScreenInfoListener {
         return true
     }
 
-    override fun onResume() {
-        super.onResume()
-        ScreenManagerProvider.getScreenManager().addScreenInfoListener(this)
-    }
-
-    override fun onPause() {
-        super.onPause()
-        ScreenManagerProvider.getScreenManager().removeScreenInfoListener(this)
+    private fun initWindowLayoutInfo() {
+        val windowInfoRepository = windowInfoRepository()
+        lifecycleScope.launch(Dispatchers.Main) {
+            lifecycle.repeatOnLifecycle(Lifecycle.State.STARTED) {
+                windowInfoRepository.windowLayoutInfo.collect { info ->
+                    onScreenInfoChanged(info)
+                }
+            }
+        }
     }
 
-    override fun onScreenInfoChanged(screenInfo: ScreenInfo) {
-        addFragments(screenInfo)
+    private fun onScreenInfoChanged(windowLayoutInfo: WindowLayoutInfo) {
```

Comment:
Same here - maybe also rename this method?

Reply:
Fixed